App.tsx

```diff
@@ -11,9 +15,14 @@ export default function App() {
     return (
       <Provider store={store}>
         <PersistGate loading={null} persistor={persistor}>
-          <Navigator />
+            <RootSiblingParent>
+                <IconRegistry icons={EvaIconsPack}/>
+                <ApplicationProvider {...eva} theme={eva.light}>
```

Comment:
Nie jestem pewien czy to jest prawidłowy sposób na robienie tego, ani czym różni się Provider od ApplicationProvider. Możliwe że Provider ma być wewnątrz ApplicationProvider (https://github.com/akveo/react-native-ui-kitten/issues/1139), albo zrezygnować z Providera w jakiś sposób na rzecz ApplicationProvidfera. Ale jak działa to można olać